### PR TITLE
WIP: Execute py tests in parallel

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: .github/actions/publish-release-contents/action.yml [hunk 1]
```diff
@@ -32,11 +32,6 @@ runs:
         cp -fR ${{ inputs.REPO_FOLDER_PATH }}/build/infrastructure/main ${{ inputs.RELEASE_FOLDER_PATH }}/infrastructure/main 2>/dev/null || :
         cp -fR ${{ inputs.REPO_FOLDER_PATH }}/build/infrastructure/env ${{ inputs.RELEASE_FOLDER_PATH }}/infrastructure/env 2>/dev/null || :
 
-    - name: Publish jobs to release folder
-      shell: bash
-      run: |
-        cp -fR ${{ inputs.REPO_FOLDER_PATH }}/source/databricks/jobs ${{ inputs.RELEASE_FOLDER_PATH }} 2>/dev/null || :
-
```
Comment (BjarkeMeier) [inline .github/actions/publish-release-contents/action.yml:39]: This is also in Dans PR.

File: .devcontainer/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: "3.9"
 services:
   wholesale:
-    image: ghcr.io/energinet-datahub/opengeh-wholesale/databricks-unit-test:latest
```
Comment (BjarkeMeier) [inline .devcontainer/docker-compose.yml:4]: Should this be committed?

File: .github/actions/publish-release-contents/action.yml (hunk 1 above)
Comment (djorgensendk) [reply .github/actions/publish-release-contents/action.yml:39]: great, thanks for pointing this out :-)

File: .devcontainer/docker-compose.yml (hunk 2 above)
Comment (djorgensendk) [reply .devcontainer/docker-compose.yml:4]: before merge, I will revert to "latest". But currently the necessary changes only exist in the docker image tagged with pre-release-pre323. After merge the latest image will be automatically updated.

**Pull request closed without merging**